pkg/api/login_test.go

```diff
@@ -207,6 +207,48 @@ func TestLoginViewRedirect(t *testing.T) {
 			appURL:      "http://localhost:3000/",
 			status:      302,
 		},
+		{
+			desc:        "non grafana url without scheme",
+			url:         "example.com",
+			redirectURL: "/",
+			appURL:      "http://localhost:3000/",
+			status:      302,
+		},
+		{
+			desc:        "non grafana url without scheme",
```

Comment:
Isn't it confusing that this has the same description as the previous case?

```suggestion
			desc:        "non-Grafana URL without scheme",
```